acknowledgements.tex

```diff
@@ -0,0 +1,13 @@
+\section{Acknowledgments}
+This work was made possible by the wonderful people in ARFC. Additionally,
+this work is funded by [my funding source].
+
+Find the appropriate acknowledgements text at
+\href{https://docs.google.com/spreadsheets/d/1VybV0oMPiqpInTwgO0ICq0EC4AL_fGj-zbXGJAhRiVU/edit?usp=sharing}{the how-to-acknowledge Google Doc}
+
+Prof. Huff is supported by the Nuclear Regulatory Commission Faculty
+Development Program (award NRC-HQ-84-14-G-0054 Program B), the Blue Waters
+sustained-petascale computing project supported by the National Science
+Foundation (awards OCI-0725070 and ACI-1238993) and the state of Illinois, the
+DOE ARPA-E MEITNER Program (award DE-AR0000983), and the DOE H2@Scale Program
+(Award Number: DE-EE0008832)
```

Comment:
This section is up-to-date, but the review from Prof. Huff makes me think that adding a note below this part saying something like, "The above section can be found in the how-to-acknowledge document, and you should check to ensure that this acknowledgment is accurate."